readme.md

```diff
@@ -73,3 +73,55 @@ bot.once('spawn', () => {
 * Dynamic harvest/mining paths
 * Sprint
 * Actual move speed per block
+
+## API
+Considering there are a lot of deep changes that are being worked on, it could take some time before it's done
+
+Also, **for now**, there is only the `pathfinder` module, `movements` and `goals` still need to be done
+
+
+# Functions:
+
+### bot.pathfinder.bestHarvestTool(block)
+Returns the best harvest tool in the inventory for the specified block
+ * `Return` - ?
+ * `block` - Block instance
+
+### bot.pathfinder.getPathTo(movements, goal, done, timeout)
+Returns a Path instance
+ * `Return` - Path instance
+ * `movments` - Movements instance
+ * `goal` - Goal instance
+ * `done` - ?
```

Comment:
this is a callback with 1 parameter, which is an object detailing the path, would be worth making some documentation for this too
https://github.com/NotSugden/mineflayer-pathfinder/blob/7ee24c1fdc620e184bf3e29a33cbbe035343104a/index.d.ts#L187-L194